ethexe/contracts/script/Deployment.s.sol

```diff
@@ -48,6 +48,7 @@ contract DeploymentScript is Script {
                         address(wrappedVara),
                         1 days,
                         2 hours,
+                        1 minutes,
```

Comment:
isn't it too small? even normal track of tx may exceed 1min

Reply:
Ok let's make it 5 minutes. Anyway after this period of time new validators will be able to sign their commitment if old validators do not committed yet.